docs/profiles.md

```diff
@@ -177,7 +175,7 @@ For example:
 
     depends:
     - name: gh-profile
-      git: username/project
+      github: username/project
       branch:  desired_branch
```

Comment:
Therefore you can use:
- nathenharvey/tmp_compliance_profile
- dev-sec/linux-baseline/tree/2.0
- dev-sec/linux-baseline/tree/48bd4388ddffde68badd83aefa654e7af3231876